Applications/Utils/FileConverter/generateMatPropsFromMatID.cpp

```diff
@@ -81,10 +86,9 @@ int main (int argc, char* argv[])
 	// set mat ids to 0 and write new msh file
 	std::fill(materialIds->begin(), materialIds->end(), 0);
```

Comment:
Now that we have property vectors it does not make sense to keep mat ids. Setting all of them to 0 was the next best thing when actually deleting mat ids was not possible. The vector should be removed now.